.travis.yml

```diff
@@ -6,7 +6,7 @@ sudo: required
 services: docker
 
 go:
-- 1.12.5
+- 1.12.7
```

Comment:
Wow..we moved so fast from 1.12.5 to 1.12.7 :-) thanks @gyuho Unless build complains anything, it lgtm